src/vs/base/browser/ui/list/listWidget.ts

```diff
@@ -978,6 +994,7 @@ export interface IListOptions<T> extends IListOptionsUpdate {
 	readonly keyboardNavigationLabelProvider?: IKeyboardNavigationLabelProvider<T>;
 	readonly keyboardNavigationDelegate?: IKeyboardNavigationDelegate;
 	readonly keyboardSupport?: boolean;
+	readonly keyboardNavigationEnabled?: boolean;
```

Comment:
@justschen This seems unnecessary, maybe lost in refactoring?

<img width="585" alt="image" src="https://github.com/microsoft/vscode/assets/22350/581f5d5b-64b0-4fb1-801f-47dadd0d4cfe">